File: src/#929/solution.go
package solution

import "strings"

func numUniqueEmails(emails []string) int {
	set := map[string]bool{}
	for _,email := range emails {
		comp := strings.Split(email, "@")
        comp[0] = strings.Replace(comp[0], ".", "", -1)
        comp[0] = strings.Split(comp[0], "+")[0]
        set[strings.Join(comp, "")] = true
	}
	return len(set)
}

File: src/#929/solution.js
var numUniqueEmails = function (emails) {
    let resEmails = emails.map((item) => {
        let res = item.split('@')
        let temp1 = res[0].replace(/\./g,'')
        temp1 = temp1.split('+')[0]
        let domain = res[1]
        return temp1 +'@'+domain
    })
    let res = new Set(resEmails)
    return res.size
}
